operation.go

```diff
@@ -69,7 +69,7 @@ func NewOperationFromString(t, v, d string) (Operation, error) {
 	if err != nil {
 		return Operation{}, fmt.Errorf("Invalid value: \"%s\"", v)
 	}
-	date, err := time.Parse("2006-01-02", d)
+	date, err := time.Parse("02/01/2006", d)
```

Comment:
No. In the line 76 of operation.go, the function UnixNano() transform the object date in the number of nanoseconds elapsed since January 1, 1970 UTC. See [https://golang.org/pkg/time/#Time.UnixNano](Time.UnixNano)